src/main/java/com/puppycrawl/tools/checkstyle/site/SiteUtil.java

```diff
@@ -151,12 +161,27 @@ public final class SiteUtil {
         // static field (all upper case)
         "SuppressWarningsHolder.aliasList",
         // loads string into memory similar to file
-        "HeaderCheck.header",
-        "RegexpHeaderCheck.header",
+        HEADERCHECK_HEADER,
+        REGEXPHEADER_CHECK_HEADER,
         // until https://github.com/checkstyle/checkstyle/issues/13376
         "CustomImportOrderCheck.customImportOrderRules"
     );
 
+    /**
+     * Frequent version.
+     */
+    private static final String V69 = "6.9";
+
+    /**
+     * Frequent version.
+     */
+    private static final String V50 = "5.0";
+
+    /**
+     * Frequent version.
+     */
+    private static final String V32 = "3.2";
+
```

Comment:
These are hard to read, please just suppress multiple string literal check for this file or add some underscores like `VERSION_3_2` and `VERSION_8_24`

Reply:
done